pkg/pgmodel/metrics/ingest.go

```diff
@@ -151,6 +151,14 @@ var (
 			Help:      "Total items (samples/exemplars/spans) received.",
 		}, []string{"type", "kind"},
 	)
+	IngestorBytes = prometheus.NewCounterVec(
```

Comment:
Since the unit is in bytes and a counter on requests may cross uint64 limit since requests are generally very big, I think it may make sense to have this is kilobytes.

Reply:
uint64 max value is 18446744073709551615, which translates to ~18446PB (Petabytes). The whole CERN generates 90PB in year, so I think we are fairly safe ;)

Reply:
I don't think so. `uint64` max is `18446744073709551615` . If for example take that we ingest 100MB/s (and that's quite  lot :) : 18446744073709551615 / ( 100000000*3600*24) = 2.1350398e+12 which seems more than enough